src/SAWScript/Interpreter.hs

```diff
@@ -2744,7 +2744,20 @@ primitives = Map.fromList
     , "`llvm_points_to`). The Term is the tuple consisting of the"
     , "output parameters of the LLVM function: the return parameter, then"
     , "the parameters passed by reference (in the order given by"
-    , "`llvm_points_to`). For more flexibility, see `llvm_verify`."
+    , "`llvm_points_to`)."
+    , ""
+    , "In order, the arguments to `llvm_compositional_extract` are:"
+    , "  1. The LLVM module containing the function to extract."
+    , "  2. The name of the function to extract."
+    , "  3. The name of the `Term` to generate."
+    , "  4. A list of overrides to use in the memory safety proof in the 7th"
+    , "     argument."
+    , "  5. Whether to perform path satisfiability checks."
+    , "  6. Memory layout specification for the extracted function."
+    , "  7. Proof that the extracted function satisfies the memory"
+    , "     specification in the 6th argument."
```

Comment:
The `ProofScript` isn't really a proof, but rather a strategy for how to compute the proof (e.g., `abc`, `z3`, `rme`, etc.)